pallets/creditcoin/src/lib.rs

```diff
@@ -655,9 +619,9 @@ pub mod pallet {
 			<T as Config>::WeightInfo::on_initialize(ask_count, bid_count, offer_count, 0, 0)
 		}
 
-		fn on_runtime_upgrade() -> Weight {
-			migrations::migrate::<T>()
-		}
+		// fn on_runtime_upgrade() -> Weight {
+		// 	migrations::migrate::<T>()
+		// }
```

Comment:
Why disable? 

We still want to have migrations regardless, no ?

Reply:
wip. to make everything compile, will reintroduce